app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -42,7 +42,7 @@ function trackGoogleAnalytics4() {
       if (!addedGA4) {
         <%# Dynamically add the Google Analytics 4 script tag %>
         var script = document.createElement('script');
-        script.src = "http://www.googletagmanager.com/gtag/js?id=" + ga4MeasurementCode;
+        script.src = "//www.googletagmanager.com/gtag/js?id=" + ga4MeasurementCode;
```

Comment:
**non-blocking question:** I truly don't know and can't test it (I don't have GA setup locally), but should the `src` here include the double slash or should it just read like this (below)?

```js
script.src = "www.googletagmanager.com/gtag/js?id=" + ga4MeasurementCode;
```

Reply:
No, we should have the `//` as prefix because that tells the browser to use the same protocol to connect the script as you have for the current page (e.g. when in https use https; when http do that instead)

Reply:
TIL. Thanks for clarifying @jeremyf!